chain/syncer_test.go

```diff
@@ -70,7 +70,7 @@ func TestTipSetIncremental(t *testing.T) {
 	verifyHead(t, store, t1)
 
 	assert.NoError(t, syncer.HandleNewTipSet(ctx, types.NewChainInfo(peer.ID(""), t2.Key(), heightFromTip(t, t2)), true))
-	verifyTip(t, store, t2, builder.StateForKey(t2.Key()))
+	// verifyTip(t, store, t2, builder.StateForKey(t2.Key()))
```

Comment:
nit: remove the commented out line here and line 151.

idea: you could also verify that `t2` is NOT in the store here.

Reply:
Test is also updated.